flake8_html/plugin.py

```diff
@@ -72,8 +75,18 @@ def after_init(self):
         self.report_template = jinja2_env.get_template('file-report.html')
         self.source_template = jinja2_env.get_template('annotated-source.html')
         if not self.options.htmldir:
-            sys.exit('--htmldir must be given if HTML output is enabled')
-        self.outdir = self.options.htmldir
+            try:
+                self.outdir = config.get('flake8', 'htmldir')
+            except:
```

Comment:
You should never write a bare except: unless it re-raises. This catches KeyboardInterrupt and SystemExit, which means your processes don't shut down correctly when Ctrl-Ced or killed.

Instead you should catch the actual errors raised, which in this case are `configparser.NoSectionError` and `configparser.NoOptionError`.